src/pfs/server/server_test.go

```diff
@@ -201,6 +209,393 @@ func TestDisallowReadsDuringCommit(t *testing.T) {
+		require.True(t, repoNames[repoInfo.Repo.Name])
+	}
+
+	require.Equal(t, len(repoInfos), numRepos-reposToRemove)
+}
+
+func TestInspectCommit(t *testing.T) {
+	t.Parallel()
+	pfsClient, _ := getClientAndServer(t)
+
+	repo := "test"
+	require.NoError(t, pfsutil.CreateRepo(pfsClient, repo))
+
+	started := time.Now()
+	commit, err := pfsutil.StartCommit(pfsClient, repo, "", "")
+	require.NoError(t, err)
+
+	fileContent := "foo\n"
+	_, err = pfsutil.PutFile(pfsClient, repo, commit.ID, "foo", 0, strings.NewReader(fileContent))
+    require.NoError(t, err)
+
+	commitInfo, err := pfsutil.InspectCommit(pfsClient, repo, commit.ID)
+	require.NoError(t, err)
+
+	require.Equal(t, commit, commitInfo.Commit)
+	require.Equal(t, pfs.CommitType_COMMIT_TYPE_WRITE, commitInfo.CommitType)
+	require.Equal(t, len(fileContent), int(commitInfo.SizeBytes))
+	require.True(t, started.Before(commitInfo.Started.GoTime()))
+	require.Nil(t, commitInfo.Finished)
+
+	require.NoError(t, pfsutil.FinishCommit(pfsClient, repo, commit.ID))
+	finished := time.Now()
+
+	commitInfo, err = pfsutil.InspectCommit(pfsClient, repo, commit.ID)
+	require.NoError(t, err)
+
+	require.Equal(t, commit, commitInfo.Commit)
+	require.Equal(t, pfs.CommitType_COMMIT_TYPE_READ, commitInfo.CommitType)
+	require.Equal(t, len(fileContent), int(commitInfo.SizeBytes))
+	require.True(t, started.Before(commitInfo.Started.GoTime()))
+	require.True(t, finished.After(commitInfo.Finished.GoTime()))
+}
+
+func TestDeleteCommit(t *testing.T) {
+	t.Parallel()
+	pfsClient, _ := getClientAndServer(t)
+
+	repo := "test"
+	require.NoError(t, pfsutil.CreateRepo(pfsClient, repo))
+
+	commit, err := pfsutil.StartCommit(pfsClient, repo, "", "")
+	require.NoError(t, err)
+
+	fileContent := "foo\n"
+	_, err = pfsutil.PutFile(pfsClient, repo, commit.ID, "foo", 0, strings.NewReader(fileContent))
+	require.NoError(t, err)
+
+	require.NoError(t, pfsutil.FinishCommit(pfsClient, repo, commit.ID))
+
+	// Because DeleteCommit is not supported
+	require.YesError(t, pfsutil.DeleteCommit(pfsClient, repo, commit.ID))
+
+	// The following code is for when DeleteCommit gets implemented
+	//t.Parallel()
+	//pfsClient, _ := getClientAndServer(t)
+
+	//repo := "test"
+	//require.NoError(t, pfsutil.CreateRepo(pfsClient, repo))
+
+	//commit, err := pfsutil.StartCommit(pfsClient, repo, "", "")
+	//require.NoError(t, err)
+
+	//fileContent := "foo\n"
+	//_, err = pfsutil.PutFile(pfsClient, repo, commit.ID, "foo", 0, strings.NewReader(fileContent))
+    //require.NoError(t, err)
+
+	//require.NoError(t, pfsutil.FinishCommit(pfsClient, repo, commit.ID))
+
+	//commitInfo, err := pfsutil.InspectCommit(pfsClient, repo, commit.ID)
+	//require.NotNil(t, commitInfo)
+
+	//require.NoError(t, pfsutil.DeleteCommit(pfsClient, repo, commit.ID))
+
+	//commitInfo, err = pfsutil.InspectCommit(pfsClient, repo, commit.ID)
+	//require.Nil(t, commitInfo)
+
+	//repoInfo, err := pfsutil.InspectRepo(pfsClient, repo)
+	//require.Equal(t, 0, repoInfo.SizeBytes)
+}
```

Comment:
Good thought leaving this around for when we need it.
I'd go 1 step forward and put this code into a separate test called `TestDeleteFuture` or something like that and then put a `t.Skip()` at the top. That way it will stay compilable as stuff changes.